### ISSUE TITLE
Failed to build successfully the ZIM file from local wiki

### ISSUE THREAD

**Issue by p-deli** [openzim/mwoffliner#500]
(This issue might be related to [this other one](https://github.com/openzim/mwoffliner/issues/499).)

Issuing following command `mwoffliner --mwUrl=http://192.168.56.5/UnifiedICTDoc --adminEmail=foo@bar.net --verbose --redis=redis://127.0.0.1:6379 --mwWikiPath=/ --mwApiPath=api.php` leads to a `Failed to build successfully the ZIM file`. The `--verbose` argument shows several things that looks odd to me:

- It seems that `zimwriterfs` expects `Main_Page.html` whereas the local wiki I have only has `Main_Page`. That is without the `.html`, when I add the extension the wiki throws the usual `There is currently no text in this page. You can search for this page title in other pages, search the related logs, or create this page.`
- It also looks that invoked `zimwriterfs` receives two `--welcome` arguments which looks odd to me (`--welcome=index.htm` and `--welcome=Main_Page.html`) as shown below:
```
zimwriterfs --welcome=index.htm --favicon=favicon.png --language=eng --welcome=Main_Page.html --verbose  --tags="56" --name="kiwix.192_en_all" --redirects="/home/pit/Downloads/cac/192_en/192_en_all.redirects" --title="Unified ICT Doc" --description="From Unified ICT Doc" --creator="192" --publisher="Kiwix" "/home/pit/Downloads/tmp/192_en_all_2019-01/" "/home/pit/Downloads/out/192_en_all_2019-01.zim")...
```

More complete `--verbose` argument output:
```
All articles were retrieved and saved.
0 files still to be downloaded.
Process still downloading images...
All images successfuly downloaded
0 images still to be optimized.
Process still optimizing images...
All images successfuly optimized
Building ZIM file /home/pit/Downloads/out/192_en_all_2019-01.zim (zimwriterfs --welcome=index.htm --favicon=favicon.png --language=eng --welcome=Main_Page.html --verbose  --tags="56" --name="kiwix.192_en_all" --redirects="/home/pit/Downloads/cac/192_en/192_en_all.redirects" --title="Unified ICT Doc" --description="From Unified ICT Doc" --creator="192" --publisher="Kiwix" "/home/pit/Downloads/tmp/192_en_all_2019-01/" "/home/pit/Downloads/out/192_en_all_2019-01.zim")...
RAID: kiwix.192_en_all
zimwriterfs: unable to find welcome page at '
/home/pit/Downloads/tmp/192_en_all_2019-01
/
Main_Page.html
'. --welcome path/value must be relative to HTML_DIRECTORY.

Failed to run mwoffliner after [105s]: "Failed to build successfuly the ZIM file /home/pit/Downloads/out/192_en_all_2019-01.zim (Error when executing zimwriterfs)"
```

----

- mwoffliner version 1.6.0
- Ubuntu release 18.10 (cosmic, desktop edition)
- `redis-server --version`: Redis server v=4.0.11 sha=00000000:0 malloc=jemalloc-5.1.0 bits=64 build=fc908f68b3e66abd

**Comment by kelson42** [comment]
@p-deli It's true that there is a bug with two `--welcome` entries, but this is only in the log message, the real command is OK. I just have it with 7f99f2beeefa534371d2207f26d90108899329e7

**Comment by kelson42** [comment]
@p-deli We have released mwoffliner 1.7 a few days ago. Have you tried it. Do you have the same problem with it?

**Comment by p-deli** [comment]
@kelson42 I am using version 1.7.1 and it works very well. Fortunately, no problems encountered so far. Thank you very much 👍
